scrapper6pm/scrapper6pm/spiders/a6pm.py

```diff
@@ -0,0 +1,87 @@
+import json
+
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+
+class A6pmSpider(CrawlSpider):
+    name = "6pm"
+    allowed_domains = ["www.6pm.com"]
+    start_urls = ["https://www.6pm.com/"]
+    data_skus = set()
+
+    rules = (
+        Rule(LinkExtractor(restrict_css='[data-sub-nav="true"] a.fi-z')),
+        Rule(LinkExtractor(restrict_css='[id="searchPagination"] a'), follow=True),
+        Rule(LinkExtractor(restrict_css='[id="products"] a[itemprop="url"]'), callback="parse_item"),
+    )
+
+    def parse_item(self, response):
+        retailer_sku = self.extract_retailer_sku(response)
+        if retailer_sku in self.data_skus:
+            return
+        item_data = {
+            "retailer_sku": self.extract_retailer_sku(response),
```

Comment:
why we are calling the function again? we have already scrapped the SKU id on line 20.